bdi_game/src/render/sdl.rs

```diff
@@ -0,0 +1,102 @@
+pub use super::traits::*;
+use sdl2;
+
+/// SDL unit
+pub struct UnitSDL<'a> {
+    pub position: &'a (i32, i32),
+    pub texture: Option<&'a sdl2::render::Texture<'a>>,
+    pub color: Option<sdl2::pixels::Color>,
```

Comment:
For the record i will write here my point that i made during our talk:
Yes, because it's fallback for us to make sure we aren't forced to use textures when we don't need to.

Reply:
I forgot how powerful are enums in rust, it will be swapped to the enum -> I will link the resolving commit here.